src/apps/material-search/MaterialSearchList.tsx

```diff
@@ -58,7 +59,7 @@ const MaterialSearchListResults: FC<MaterialSearchListResultsProps> = ({
       </div>
       <ol className="material-search-list__results">
         {data.map((work, index) => {
-          const authors = flattenCreators(work.creators);
+          const authors = flattenCreators(work.creators as Work["creators"]);
```

Comment:
I think we should try to infer the `Work` type as early as possible so we do not need to use type assertions later on.
Please look at this [suggestion](https://stackblitz.com/edit/vitejs-vite-hzqbpm?file=src%2FMaterialSearchList.tsx%3AL54&terminal=dev)

Reply:
That's a great way to do it! thx for the suggestion ✨